site/docs/es/plugins/parse-mode.md

```diff
@@ -2,28 +2,138 @@
 
 Este plugin proporciona un transformador para establecer el `parse_mode` por defecto, y un middleware para hidratar el `Context` con los métodos variantes familiares de `reply` - es decir, `replyWithHTML`, `replyWithMarkdown`, etc.
 
-## Uso
+## Uso (Mejorar la experiencia de formateo)
 
 <CodeGroup>
   <CodeGroupItem title="TypeScript" active>
 
 ```ts
-import { Bot } from "grammy";
+import { Bot, Context } from "grammy";
+import { bold, fmt, hydrateReply, italic, link } from "@grammyjs/parse-mode";
+
+import type { ParseModeFlavor } from "@grammyjs/parse-mode";
+
+const bot = new Bot<ParseModeFlavor<Context>>("");
+
+// Instalar el plugin.
+bot.use(hydrateReply);
+
+bot.command("demo", async (ctx) => {
+  await ctx.replyFmt(fmt`${bold("negrita!")}
+${bold(italic("ambas!"))}
+${bold(fmt`negrita ${link("blink", "example.com")} negrita`)}`);
+
+  // fmt también puede ser llamada como cualquier otra función.
+  await ctx.replyFmt(
+    fmt(
+      ["", " and ", " and ", ""],
```

Comment:
I think we can safely translate these words.

```suggestion
      ["", " y ", " y ", ""],
```

Reply:
Here is the result for this snippet

![32c12xe1cex1q](https://user-images.githubusercontent.com/74030149/210091117-bde4bcef-99d1-4de9-ae50-4ad1cefee0d4.jpg)

Reply:
Yes, they should be translated. This is part of a message string, and you usually translate those, as well :)